File: proyectomds2/frontend/src/vista-crear-lista-reproduccion.js
import {html, PolymerElement} from '@polymer/polymer/polymer-element.js';
import '@vaadin/vaadin-ordered-layout/src/vaadin-vertical-layout.js';
import '@vaadin/vaadin-ordered-layout/src/vaadin-horizontal-layout.js';
import '@vaadin/vaadin-text-field/src/vaadin-text-field.js';
import '@vaadin/vaadin-button/src/vaadin-button.js';

class VistaCrearListaReproduccion extends PolymerElement {

    static get template() {
        return html`
<style include="shared-styles">
                :host {
                    display: block;
                    height: 100%;
                }
            </style>
<vaadin-vertical-layout style="width: 100%; height: 100%;" id="vaadinVerticalLayout">
 <vaadin-text-field label="Titulo lista reproduccion" placeholder="Placeholder" style="width: 90%;" id="text_field_lista_reproduccion"></vaadin-text-field>
 <vaadin-horizontal-layout theme="spacing" style="width: 100%;" id="vaadinHorizontalLayout">
  <vaadin-text-field label="Cancion" placeholder="Placeholder" style="width: 100%;" id="text_field_cancion"></vaadin-text-field>
  <vaadin-button style="width: 10%; align-self: flex-end;" id="vaadinButton" dir="boton_buscar">
    Buscar 
  </vaadin-button>
 </vaadin-horizontal-layout>
 <label id="label">Lista de reproduccion</label>
 <vaadin-vertical-layout theme="spacing" style="width: 100%; height: 100%;" id="lista_reproduccion"></vaadin-vertical-layout>
 <vaadin-button style="align-self: flex-end;" id="boton_finalizar">
   Finalizar 
 </vaadin-button>
</vaadin-vertical-layout>
`;
    }

    static get is() {
        return 'vista-crear-lista-reproduccion';
    }

    static get properties() {
        return {
            // Declare your properties here.
        };
    }
}

customElements.define(VistaCrearListaReproduccion.is, VistaCrearListaReproduccion);

File: proyectomds2/src/main/java/vistas/VistaCrearListaReproduccion.java
package vistas;

import com.vaadin.flow.templatemodel.TemplateModel;
import com.vaadin.flow.component.Tag;
import com.vaadin.flow.component.dependency.JsModule;
import com.vaadin.flow.component.polymertemplate.PolymerTemplate;
import com.vaadin.flow.dom.Element;
import com.vaadin.flow.component.template.Id;
import com.vaadin.flow.component.textfield.TextField;
import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
import com.vaadin.flow.component.button.Button;
import com.vaadin.flow.component.html.Label;

/**
 * A Designer generated component for the vista-crear-lista-reproduccion template.
 *
 * Designer will add and remove fields with @Id mappings but
 * does not overwrite or otherwise change this file.
 */
@Tag("vista-crear-lista-reproduccion")
@JsModule("./src/vista-crear-lista-reproduccion.js")
public class VistaCrearListaReproduccion extends PolymerTemplate<VistaCrearListaReproduccion.VistaCrearListaReproduccionModel> {

    @Id("vaadinVerticalLayout")
	private Element vaadinVerticalLayout;
	@Id("text_field_lista_reproduccion")
	private TextField text_field_lista_reproduccion;
	@Id("vaadinHorizontalLayout")
	private HorizontalLayout vaadinHorizontalLayout;
	@Id("text_field_cancion")
	private TextField text_field_cancion;
	@Id("vaadinButton")
	private Button vaadinButton;
	@Id("label")
	private Label label;
	@Id("lista_reproduccion")
	private Element lista_reproduccion;
	@Id("boton_finalizar")
	private Button boton_finalizar;

	/**
     * Creates a new VistaCrearListaReproduccion.
     */
    public VistaCrearListaReproduccion() {
        // You can initialise any data required for the connected UI components here.
    }

    /**
     * This model binds properties between VistaCrearListaReproduccion and vista-crear-lista-reproduccion
     */
    public interface VistaCrearListaReproduccionModel extends TemplateModel {
        // Add setters and getters for template properties here.
    }
}
